src/DynamicMvvm.Abstractions/DynamicMvvm.Abstractions.csproj

```diff
@@ -9,10 +9,19 @@
 		<PackageId>Chinook.DynamicMvvm.Abstractions</PackageId>
 		<Description>Chinook.DynamicMvvm.Abstractions</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
 	</PropertyGroup>
 
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
 	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
```

Comment:
Could those be in the same item group as the other package references?

Reply:
I don't see any mentions that it wouldn't work.